src/ResourceManager/EventHub/Commands.EventHub.Test/packages.config

```diff
@@ -5,15 +5,15 @@
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="1.0.2" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.EventHub" version="2.0.3" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.EventHub" version="2.1.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Websites" version="1.2.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Test.HttpRecorder" version="1.7.0" targetFramework="net45" />
   <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
-  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net452" />
-  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net45" />
```

Comment:
The only line of change should be to Eventhub package - revert these changes.

Reply:
reverted